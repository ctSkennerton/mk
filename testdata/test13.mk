# Assorted expansion types.
targets = rupert ruxpin
targetpath = ./testdata
joinedbincmds = ${targets:%=$targetpath/%}
suffix = o
prefix = name
suffixed = ${targets:%=%.$suffix}
suffixes = o ab d
multisuffixed = ${targets:%=%.$suffixes}

# all:V: $joinedbincmds
all:V: 
	process1 $joinedbincmds
	process2 ab$targetpath
	process3 $targetpathab
	process4 $targetpath/foo
	process5 ${targetpath}/foo
	process6 "$targetpath"
	process7 $prefix.$suffix
	process8 $suffixed
	process9 $multisuffixed

