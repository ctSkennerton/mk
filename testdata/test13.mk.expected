all: process1 ./testdata/rupert ./testdata/ruxpin
      process2 ab./testdata
      process3 $targetpathab
      process4 ./testdata/foo
      process5 ./testdata/foo
      process6 "./testdata"
      process7 name.o
      process8 rupert.o ruxpin.o
      process9 rupert.o ab d ruxpin.o ab d
